insights/parsers/ethtool.py

```diff
@@ -815,31 +823,16 @@ def parse_content(self, content):
                 except:
                     raise ParseException('Ethtool unable to parse content', line)
 
-        self._supported_link_modes = []
+        self.supported_link_modes = []
```

Comment:
Please add their description to the class's docsting.

Reply:
Hi @xiangce - sorry for the delay, I was away.  The attributes in this are badly documented, so I'm going to document them throughout.